server/s3_bucket_metadata_cache.h

```diff
@@ -51,24 +54,29 @@ class S3BucketMetadataCache {
 
   virtual ~S3BucketMetadataCache();
 
+  static S3BucketMetadataCache* get_instance();
+
   void disable() { disabled = true; }
   void enable() { disabled = false; }
 
   using FetchHanderType =
```

Comment:
have comments on it and some other places also

Reply:
Will add.